RELEASING.md

```diff
@@ -1,77 +1,93 @@
+
+---
 
 ## Release Candidates
 
-For significant API changes that could result in significant refactoring on the part of developers, consider releasing a few _Release Candidate_ versions in advance of the final version. `shipit` is configured to do this from the `shopify_api_next` branch.
+For significant API changes that could result in significant refactoring on the part of developers, consider releasing a few _Release Candidate_ versions in advance of the final version.
 
-1. Ensure your local repo is up-to-date
+> **Warning**
+>
+> These changes **must** be made against the `next` branch, so that the appropriate workflows can run (`next-release.yml`).
 
-   ```shell
-   git checkout shopify_api_next && git pull
-   ```
+> **Warning**
+>
+> Before commencing the effort for a batch of release candidates, make sure the `next` branch an identical copy of `main`.
 
-1. (optional) Add an entry for the release candidate to `CHANGELOG.md`
+- Prior to creating the first PR against the `next` branch, run the `yarn changeset pre enter rc` command and commit the resultant files from `.changeset`, including the `pre.json` file. This informs `changesets` that it is in pre-release mode, and the pre-release tag is `rc`.
 
-1. Increment the version in `lib/version.ts`, ensuring that it ends with `-rcN`, where `N` starts at `1` and increments with each Release Candidate.
+- When creating a PR, the author should run the `yarn changeset` command, answer the relevant questions (i.e., is it major/minor/patch, what is the change description), and then commit the new file created in the `.changeset` directory. These files are used by the workflows to construct the `CHANGELOG.md` entries for the release candidates.
 
-1. Update the version in `package.json` to match that in `lib/version.ts`
+  > **Note**
+  > If the change is very small and doesn't warrant a changelog entry, run `yarn changeset --empty` and commit the resultant file in the `.changeset` directory.
 
-1. Stage the `CHANGELOG.md`, `lib/version.ts`, and `package.json` files
+- When the PR is merged into the `next` branch, the `next-release.yml` workflow uses the `changesets/action` to either create or update an existing PR that has the title `Version Packages for Release Candidates`.
 
-   ```shell
-   git add CHANGELOG.md lib/version.ts package.json
-   ```
+### :exclamation: To perform a release of release candidate packages
 
-1. Commit the files to the repo
+1. Checkout the `changeset-release/next` branch
 
    ```shell
-   git commit -m "Packaging for release vX.Y.Z-rcN"
+   git checkout changeset-release/next
    ```
 
-1. Tag the commit with an _annotated_ tag (required for the `--follow-tags` of the `push` command). Note that it's very important that the tag includes `-rc` as part of the tag name - this is how `shipit` will recognize that it's a release candidate, not a general release.
+1. Update the version string in the `lib/version.ts` file to match the version in the `package.json` file.
```

Comment:
I think we'll also need to update this file with the appropriate RC version, and changesets is a bit prescriptive about how it should be done, right?

```suggestion
1. Update the version string in the `lib/version.ts` file to match the version in the `package.json` file. Make sure to add an `-rc.X` to the version, like so:

``
7.0.0-rc.1
``

```

Reply:
I Agree!